test/javascripts/components/tc-message-collapser-test.js

```diff
@@ -0,0 +1,145 @@
+import componentTest, {
+  setupRenderingTest,
+} from "discourse/tests/helpers/component-test";
+import hbs from "htmlbars-inline-precompile";
+import {
+  discourseModule,
+  exists,
+  query,
+} from "discourse/tests/helpers/qunit-helpers";
+
+discourseModule(
+  "Discourse Chat | Component | tc message collapser",
+  function (hooks) {
+    setupRenderingTest(hooks);
+    const youtubeCooked =
+      '<div class="onebox lazyYT lazyYT-container" data-youtube-id="WaT_rLGuUr8" data-youtube-title="Japanese Katsu Curry (Pork Cutlet)"/>';
+    const imageCooked = "<p>A picture of Tomtom</p>";
+
+    // youtube
+    componentTest("shows youtube link in header", {
```

Comment:
I might be wrong on this, but my feeling is this text is supposed to define context not the expected output.

Expected output can be written down in last parameter of the assertions:

```
assert.ok(true, "it shows the truth")
```

Reply:
qunit makes it weird to define context since it doesn't have the nice `context` block to wrap around a set of component tests. Ideally the ones after `//youtube` are all for youtube and the ones after `//images` are all for images.

Reply:
that said maybe I can extract them into two separate files and have the `discourseModule` description say so

Reply:
You can create multiple acceptance() blocks for this, agreed this is more complicated than rspec but 🤷‍♂️